Source/Core/Core/HW/Memmap.cpp

```diff
@@ -286,6 +287,7 @@ u8* GetPointer(const u32 address)
 			if ((address & 0xfffffff) < REALRAM_SIZE)
 				return m_pRAM + (address & RAM_MASK);
 		}
+		break;
```

Comment:
Actually, pinging @magumagu. Should this be a marked fallthrough, or would breaking here be correct?